core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -226,23 +226,16 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     gcsDAO.listBillingAccounts(userInfo) map(RequestComplete(_))
 
   def getBillingProjectMembership(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = {
-    val membershipsFuture = for {
-      resourceIdsWithPolicyNames <- samDAO.getPoliciesForType(SamResourceTypeNames.billingProject, userInfo)
-      projectDetailsByName <- dataSource.inTransaction { dataAccess => dataAccess.rawlsBillingProjectQuery.getBillingProjectDetails(Set(projectName)) }
+    val membershipFuture = for {
+      resourceIdWithPolicyName <- samDAO.getResourcePolicies(SamResourceTypeNames.billingProject, projectName.toString, userInfo)
```

Comment:
You don't want `projectName.toString` because it takes a `RawlsBillingProjectName`

Reply:
For some reason, this sam method doesn't work for me. For cases where I'm a billing project user, it gives a 403. For cases where I'm an owner, it comes back with an empty list. I'll dig in a little more.